### PR TITLE
fix error in world builder for unity build

### PR DESCRIPTION
**Pull request by tjhei**
reported in #3725 and #3692

### CODE REVIEW

**Comment by tjhei** [comment]
Let me double-check that this now compiles on my branch...

**Comment by tjhei** [comment]
It is compiling for me now.

File: contrib/world_builder/source/features/mantle_layer.cc [hunk 1]
```diff
@@ -138,7 +138,7 @@ namespace WorldBuilder
                                                                       *(world->parameters.coordinate_system));
 
       if (depth <= max_depth && depth >= min_depth &&
-          Utilities::polygon_contains_point(coordinates, Point<2>(natural_coordinate.get_surface_coordinates(),
+          WorldBuilder::Utilities::polygon_contains_point(coordinates, Point<2>(natural_coordinate.get_surface_coordinates(),
                                                                   world->parameters.coordinate_system->natural_coordinate_system())))
```
Comment (bangerth) [inline contrib/world_builder/source/features/mantle_layer.cc:142]: I think you probably need to indent that.

Comment (tjhei) [reply contrib/world_builder/source/features/mantle_layer.cc]: Yep. Figured out I can run the worldbuilder indent. :-)

**Pull request merged**